### DISCUSSION TITLE
How can I test locally whether it is deployed or not?

### DISCUSSION THREAD

**Discussion by aditmehta9** [JamesIves/github-pages-deploy-action#617] [Q&A] [answered]
I am new in Github Actions I wanted to know how can I know if it is successfully deployed?
Here's the code:
`
# Basic workflow for Actions

name: Build and Deploy to GitHub static pages
# Controls when the action will run. 
on:
  # Triggers the workflow on push or pull request events but only for the develop branch
  push:
    branches: [ develop ]

  # Allows you to run this workflow manually from the Actions tab
  workflow_dispatch:

# A workflow run is made up of one or more jobs that can run sequentially or in parallel
jobs:
  # This workflow contains a single job called "build"
  build:
    # The type of runner that the job will run on
    runs-on: ubuntu-latest

    # Steps represent a sequence of tasks that will be executed as part of the job
    steps:
      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it

      - name: Checkout 🛎️
        uses: actions/checkout@v2.3.1 # If you're using actions/checkout@v2 you must set persist-credentials to false in most cases for the deployment to work correctly.
        with:
          persist-credentials: false

      - name: Install and Build 🔧 # This example project is built using npm and outputs the result to the 'build' folder. Replace with the commands required to build your project, or remove this step entirely if your site is pre-built.
        run: |
          npm install
          npm run build
      - name: Deploy to Github Pages
        uses: JamesIves/github-pages-deploy-action@3.7.1
        with:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          BRANCH: master # The branch the action should deploy to.
          FOLDER: build # The folder the action should deploy.
          CLEAN: true # Automatically remove deleted files from the deploy branch
`
Screenshot after adding in workflow folder:
![image](https://user-images.githubusercontent.com/35133994/107322791-26fdba80-6acb-11eb-8b23-8f0c89c9ec23.png)

**Comment by JamesIves** [comment, marked as answer]
I'm not really sure what you're asking here, are you asking if you can run your workflow locally?

**Reply by aditmehta9** [reply]
> I'm not really sure what you're asking here, are you asking if you can run your workflow locally?

I am sorry for the unclear question however it is fixed. I just wanted to know how can I deploy the frontend because whenever I deploy it deploys README instead of the frontend of the website.
